sdk/securityinsight/arm-securityinsight/CHANGELOG.md

```diff
@@ -1,15 +1,233 @@
+  - Interface BookmarkRelationsListNextOptionalParams no longer has parameter orderby
+  - Interface BookmarkRelationsListNextOptionalParams no longer has parameter skipToken
+  - Interface BookmarkRelationsListNextOptionalParams no longer has parameter top
+  - Interface EntitiesRelationsListNextOptionalParams no longer has parameter filter
+  - Interface EntitiesRelationsListNextOptionalParams no longer has parameter orderby
+  - Interface EntitiesRelationsListNextOptionalParams no longer has parameter skipToken
+  - Interface EntitiesRelationsListNextOptionalParams no longer has parameter top
+  - Interface EntityQueriesListNextOptionalParams no longer has parameter kind
+  - Interface FileImportsListNextOptionalParams no longer has parameter filter
+  - Interface FileImportsListNextOptionalParams no longer has parameter orderby
+  - Interface FileImportsListNextOptionalParams no longer has parameter skipToken
+  - Interface FileImportsListNextOptionalParams no longer has parameter top
+  - Interface IncidentCommentsListNextOptionalParams no longer has parameter filter
+  - Interface IncidentCommentsListNextOptionalParams no longer has parameter orderby
+  - Interface IncidentCommentsListNextOptionalParams no longer has parameter skipToken
+  - Interface IncidentCommentsListNextOptionalParams no longer has parameter top
+  - Interface IncidentRelationsListNextOptionalParams no longer has parameter filter
+  - Interface IncidentRelationsListNextOptionalParams no longer has parameter orderby
+  - Interface IncidentRelationsListNextOptionalParams no longer has parameter skipToken
+  - Interface IncidentRelationsListNextOptionalParams no longer has parameter top
+  - Interface IncidentsListNextOptionalParams no longer has parameter filter
+  - Interface IncidentsListNextOptionalParams no longer has parameter orderby
+  - Interface IncidentsListNextOptionalParams no longer has parameter skipToken
+  - Interface IncidentsListNextOptionalParams no longer has parameter top
+  - Interface MetadataListNextOptionalParams no longer has parameter filter
+  - Interface MetadataListNextOptionalParams no longer has parameter orderby
+  - Interface MetadataListNextOptionalParams no longer has parameter skip
+  - Interface MetadataListNextOptionalParams no longer has parameter top
+  - Interface ThreatIntelligenceIndicatorsListNextOptionalParams no longer has parameter filter
+  - Interface ThreatIntelligenceIndicatorsListNextOptionalParams no longer has parameter orderby
+  - Interface ThreatIntelligenceIndicatorsListNextOptionalParams no longer has parameter skipToken
+  - Interface ThreatIntelligenceIndicatorsListNextOptionalParams no longer has parameter top
+  - Interface WatchlistItemsListNextOptionalParams no longer has parameter skipToken
+  - Interface WatchlistsListNextOptionalParams no longer has parameter skipToken
+  - Parameter severity of interface Incident is now required
```

Comment:
line 100,101,102 are the new breaking changes when adding "required": ["properties"] in swagger